src/main/java/com/github/edouardswiac/zerotier/ZTService.java

```diff
@@ -25,4 +30,17 @@
   ZTNetworkMember getNetworkMember(String networkId, String address);
   void updateNetworkMember(ZTNetworkMember networkMember);
   void deleteNetworkMember(String networkId, String address);
+
+  List<ZTNet> getNets();
```

Comment:
I think the service API should be explicit, and I would prefer if you could use long form naming, like `getNetworks`, `getControllerNetwork` etc...